.github/workflows/release.yml

```diff
@@ -0,0 +1,39 @@
+name: Releases
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: Bump Version
+        default: v0.0.1
+        required: true
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up Go 1.16
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.16
+
+      - name: Check out source code
+        uses: actions/checkout@v2
+        with:
+          path: main
+          ref: ${{ github.event.pull_request.head.sha }}
+
+      - name: Tidy
+        env:
+          GOPROXY: "https://proxy.golang.org"
+        run: cd main && go mod tidy && git diff --quiet HEAD
+
+      - name: Build openshift plugin
+        env:
+          GOPROXY: "https://proxy.golang.org"
+        run: cd main/ && GOOS=linux GOARCH=amd64 go build -o ~/main/bin/OpenshiftPlugin-${{ github.event.inputs.version }} .
```

Comment:
Do we need to build for more than just linux/amd64?

Reply:
We need to figure out which ones we want to provide the binaries for.